lib/rubygems/spec_fetcher.rb

```diff
@@ -171,30 +171,34 @@ def spec_for_dependency(dependency, matching_platform=true)
   # Suggests gems based on the supplied +gem_name+. Returns an array of
   # alternative gem names.
 
-  def suggest_gems_from_name(gem_name, type = :latest)
+  def suggest_gems_from_name(gem_name, type = :latest, num_results = 5)
     gem_name        = gem_name.downcase.tr('_-', '')
+    name            = Regexp.new gem_name
```

Comment:
If you apply my suggestion to use `.include?(...)` below, be sure to remove this line since it will be unused. :slightly_smiling_face: 

```suggestion
```